src/i18n/pl/index.ts

```diff
@@ -64,6 +64,7 @@ export default {
       title: 'Stwórz projekt',
       projectName: 'Nazwa projektu',
       description: 'Opis (opcjonalne)',
+      tags: 'Tagi (optional)',
```

Comment:
czemu wszystko po polsku a to po angielsku?

Reply:
Zapomniałem zmienić, naprawione :D